tests/tests/tier0/bluechi-is-online-system-wait/test_bluechi_is_online_system_wait.py

```diff
@@ -0,0 +1,205 @@
+#
+# Copyright Contributors to the Eclipse BlueChi project
+#
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+import logging
+import threading
+import time
+from typing import Dict
+
+from bluechi_test.bluechi_is_online import BluechiIsOnline
+from bluechi_test.config import BluechiAgentConfig, BluechiControllerConfig
+from bluechi_test.machine import BluechiAgentMachine, BluechiControllerMachine
+from bluechi_test.test import BluechiTest
+from bluechi_test.util import Timeout, get_test_env_value_int
+
+LOGGER = logging.getLogger(__name__)
+
+AGENT_ONE = "agent-one"
+AGENT_TWO = "agent-two"
+IMMEDIATE_RETURN_TIMEOUT_MS = get_test_env_value_int("IMMEDIATE_RETURN_TIMEOUT", 1000)
```

Comment:
I think the name `IMMEDIATE` is a bit misleading. I'd prefer something like `SHORT_WAIT_PARAM_VALUE_MS`.

Reply:
This name was already agreed on in https://github.com/eclipse-bluechi/bluechi/pull/1008#discussion_r1888065496
so I thought it would be nice to align it with other tests.
But if you think it's necessary I'll change it

Reply:
I think it describes nicely the constant meaning

Reply:
Would still prefer something else than immediate (i.e. a "right now" timeout seems off), but lets keep it for consistency sake.